Src/DependencyCollector/Shared/Implementation/DependencyCollectorEventSource.cs

```diff
@@ -267,8 +267,12 @@ private string GetApplicationName()
             try
             {
 #if NETCORE
+#if NETSTANDARD1_3
+                name = "Undefined"; // netstandard1.3 does not have API to retrieve Assembly/AppDomain name
```

Comment:
there is nothing in netstandard 1.3 that could be used. Not even a process.

You can check it youself: [netstandard1.3 API list](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard1.3_ref.md)